aws/lambda_test.go

```diff
@@ -0,0 +1,158 @@
+package aws
+
+import (
+	"archive/zip"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/lambda"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+
+	"github.com/gruntwork-io/cloud-nuke/util"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
The following three functions are general-purpose utilities for working with zip files, so ideally we would have them live in the `util` package. I don't want to block merging on this, so if you just added a TODO explaining that these functions should be moved over to the util package, that would be great.

Reply:
Thanks for the heads up @infraredgirl . I have added the "TODO" comments as you have suggested 👍 
If I find some time this weekend, I will move these into the "utils" package.

Reply:
I'll be merging this since it's not a blocker! Create a new PR with the changes when you have time.